### ISSUE TITLE
Spots fail to refresh automatically

### ISSUE THREAD

**Issue by minitajfun** [coulisse/spiderweb#28]
**Describe the bug**
The table with spots does not refresh automatically. They can still be refreshed manually by entering the page again.

**To Reproduce**
1. Go to the page
2. Wait
3. Spots do not refresh

**Additional info:**
 - OS: Ubuntu 22.04
 - Program ver: [ea410729eb31c6a81ce79b959040fee30f2192bb](https://github.com/coulisse/spiderweb/tree/ea410729eb31c6a81ce79b959040fee30f2192bb)
 - Python ver: 3.10.6
 - There seems to be an issue in the Javascript code of table.js, as it returns this error:
```
Uncaught TypeError: can't access property "selectedOptions", document.getElementById(...) is null
    getFilter http://127.0.0.1:8080/static/js/table.min.js:5
    myTimer http://127.0.0.1:8080/static/js/table.min.js:6
```

**Comment by coulisse** [comment]
Hello
I couldn't reproduce the error.

It seems a problem related to filters.

Have you public url in order to make a  check?

Could you also give me: 
- your config files in CFG folder (remove your db user and password!) 
- your webapp.log 


Thanks
Corrado - IU1BOW

**Comment by minitajfun** [comment]
[cfg.zip](https://github.com/coulisse/spiderweb/files/10354159/cfg.zip)
[webapp.log](https://github.com/coulisse/spiderweb/files/10354162/webapp.log)
[website](http://sq3mzm.ddns.net:8080/)

**Comment by coulisse** [comment]
Hello
in your **config.json**, try to change 

`"enable_cq_filter":"n",`
to
`"enable_cq_filter":"y",`

If it is ok, I will fix this error in the 2.4.1 version.

Waiting for your feedback.
Best regards
Corrado

**Comment by minitajfun** [comment]
Changed `enable_cq_filter`, now spots refresh as they should.
Thanks for the help!
